### ISSUE TITLE
Allow HTTP connections

### ISSUE THREAD

**Issue by cgrima** [linuxserver/docker-pydio#3]
Would be interesting to let the port 80 opened or to have a selector when running the container to allow for HTTP connections instead of "HTTPS only". Some people like me would need to use HTTP because the SSL part is already supported by a reverse-proxy upfront.

I guess that was possible before commit ["make HTTPS only"](https://github.com/linuxserver/docker-pydio/commit/1a2f67347c27db363dfbd57ddaca146be0e70b8a)

**Comment by sparklyballs** [comment]
by changing the file /config/nginx/site-confs/default you can make it http if you like.

**Comment by cgrima** [comment]
Thank you Sparklyballs. I have used the default configuration file as before the commit, the one with the "listen 80" server section added.

HTTPS connection works but an HTTP connection redirects me to the following url [https://_](https://_) (yes, with an underscore at the end).  This happens either from inside our outside my network. Any idea why?

I'm not a pro of nginx configuration files. I've tried to change the line `server_name _;` into something like `server_name www.mydomain.com;`, then restart the container, but get the same error.
